blazar_dashboard/static/leases/js/create_lease/extra_capability_widget.js

```diff
@@ -16,6 +16,9 @@ function capabilitiesjs(resource_type, switchable_classname) {
     xhr.onreadystatechange = function() {
         if (this.readyState == 4 && this.status == 200) {
             capabilityValues = JSON.parse(this.responseText)['extra_capabilities'];
+            if (capabilityValues == undefined) {
```

Comment:
This fixes a non-critical exception that would get logged sometimes. Not essential to this change.